modules/wpml/wpml-api.php

```diff
@@ -218,24 +218,26 @@ public static function wpml_switch_language( $lang = null, $cookie = false ) {
 	 *
 	 * @since 2.0
 	 *
-	 * @param mixed $language_code
-	 * @param array $args          An array with two keys element_id => post_id or term_taxonomy_id, element_type => post type or taxonomy
+	 * @param string $language_code A 2-letter language code.
+	 * @param array  $args          An array with two keys element_id => post_id or term_taxonomy_id, element_type => post type or taxonomy
 	 * @return string
```

Comment:
```suggestion
	 * @return string|null
```
Even if it's not exactly what is document, it seems to be what WPML really returns.

Reply:
Done in [7b2e657](https://github.com/polylang/polylang/pull/1207/commits/7b2e6577fdfd7577c7a15bd94a674c9b66487671).